hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java

```diff
@@ -322,8 +322,16 @@ public DBUpdatesWrapper getUpdatesSince(long sequenceNumber, long limitCount)
         }
         transactionLogIterator.next();
       }
+    } catch (SequenceNumberNotFoundException e) {
```

Comment:
Should revert back to throwing all IOExceptions?

Reply:
I believe `SequenceNumberNotFoundException` is the only expected checked exception to be thrown anyway, judging from the method signature:

https://github.com/apache/ozone/blob/946604705cd3a84888f3d189b1938f72f508b664/hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java#L280-L281